web/themes/custom/pineapple/sass/components/paragraphs/_image-paragraph.scss

```diff
@@ -1,14 +1,23 @@
 .paragraph--type--image-paragraph {
-  .image-paragraph-image {
-    img {
-      object-fit: cover;
-      width: 100%;
+  .inline-field-label {
+
+    .field__label {
+      font-weight: bold;
+      margin-left: -$unit;
+      font-size: $small-font;
+    }
+
+    .field__value {
+      font-size: $small-font;
     }
   }
+}
+
+.field--type-image {
+  width: 50%;
```

Comment:
Consider this being 100% on mobile because it's too small as it is.

Reply:
Done